### PR TITLE
Feature: Export Issues from a Project as a CSV

### PR DESCRIPTION
**Pull request by Bracciata**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "monitor-utility",
-  "version": "2.0.3",
-  "description": "Utility for verbose outputs of Monitor data using a CLI",
```
Comment (dequeMai) [inline package.json:4]: I don't think this is an intended change.

File: src/index.js [hunk 2]
```diff
@@ -12,14 +14,6 @@ const year = date.getFullYear();
 
 console.log("\nMonitor Utility \nVersion: " + version);
 
-console.log(`
---------------------------------------------------
```
Comment (dequeMai) [inline src/index.js:16]: This is still true for exporting projects only, right?

File: src/index.js [hunk 3]
```diff
@@ -52,29 +46,83 @@ inquirer
...
-    reporter(answers);
+    if (answers.path === "issues") {
+      // Prompt for the project ID and then export all issues.
+      inquirer
+        .prompt([
+          {
+            type: "input",
+            name: "projectid",
+            message:
+              "What are the Project Ids (the *** in {Axe Monitor URL}/worldspace/issues/*** separated by commas(,))?",
```
Comment (dequeMai) [inline src/index.js:75]: Could we change language here to be more clear?

"List all Project Id(s) you'd like to export, separated by a comma (i.e. 24,15,215). Project Id can be found in your Axe Monitor URL. 

Example:  {insert example url}"

Comment (dequeMai) [inline src/index.js:75]: Off comment ... would be nice to have a feature where we validate if the project id actually exists or not.

**Review by dequeMai — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Bracciata) [reply package.json:4]: Good catch. Only version was intended

File: src/index.js (hunk 2 above)
Comment (Bracciata) [reply src/index.js:16]: That is correct. I intended to move it lower and forgot. Doing that now.

Comment (Bracciata) [reply src/index.js:16]: Correction. It was moved lower.  This way it only shows if you are exporting projects (not when exporting issues).

File: src/index.js (hunk 3 above)
Comment (Bracciata) [reply src/index.js]: That is a great idea! I am going to add that to the backlog though if that is okay!

Comment (dequeMai) [reply src/index.js]: For sure.

**Pull request merged**